src/index.ts

```diff
@@ -1042,19 +1045,21 @@ export class ValidationService {
     /**
      * Load default validation providers and scans the entire document when ready.
      * @param options.watch If set to true, a MutationObserver will be used to continuously watch for new elements that provide validation directives.
+     * @param options.addNoValidate If set to true (the default), a novalidate attribute will be added to the containing form in validate elemets.
      */
-    bootstrap(options?: { watch?: boolean, root?: HTMLElement }) {
+    bootstrap(options?: { watch?: boolean, root?: HTMLElement, addNoValidate?: boolean }) {
         options = options || {};
 
         this.addMvcProviders();
         let document = window.document;
         const root = options.root || document.body;
+        const addNoValidate = options.addNoValidate || true;
         const init = () => {
-            this.scan(root);
+            this.scan(root, addNoValidate);
```

Comment:
I think this can be simplified quite a bit:

```suggestion
            this.scan(root);
            if (addNoValidate) {
              for (var form of this.formInputs) {
                form.setAttribute('novalidate', 'novalidate');
              }
            }
```

The `scan` method builds up a list of all the relevant `form` inputs. We can rely on that after `scan` is done.